osu.Game/Screens/Edit/Screens/Compose/Layers/HitObjectMaskLayer.cs

```diff
@@ -33,23 +33,29 @@ private void load()
         {
             maskContainer = new MaskContainer();
 
-            selectionBox = composer.CreateSelectionBox(maskContainer);
+            maskSelection = composer.CreateSelectionBox(maskContainer);

```

Comment:
While we're here, how about we rename the composer method too? Hasn't been a selection box since the beginning of this PR :P